common/src/main/resources/ts/models/FormElement/FormElement.ts

```diff
@@ -49,6 +49,7 @@ export abstract class FormElement implements Selectable {
     }
 
     equals = (formElement: FormElement) : boolean => {
+        if (!formElement) return false;
         return this.form_element_type === formElement.form_element_type && this.id === formElement.id;
```

Comment:
return this.form_element_type === formElement.form_element_type && this.id === formElement.id;
```suggestion
        return formElement && this.form_element_type === formElement.form_element_type && this.id === formElement.id;
```